src/index.js

```diff
@@ -17,4 +18,12 @@ if (process.env.NODE_ENV === 'development') {
   });
 }
 
-ReactDOM.render(<App />, document.getElementById('root'));
+ReactDOM.render(
+  <Router>
+    <Switch>
+      <Route exact path="/login" render={() => <Auth />}></Route>
+      <Route path="/" render={() => <App />}></Route>
```

Comment:
Personally I think the landing page should be `/`. That way when you just go directly to the base url you are prompted with the splash screen